### ISSUE TITLE
Support for Bamboo 8.x

### ISSUE THREAD

**Issue by lancelinksp** [jfrog/bamboo-artifactory-plugin#151]
8.0 was released on July 28th and it is now September 15th. How long before this is compatible? We are now coming up on 2 months since release of Bamboo 8.0.0 which was about 9 days prior to your last release.
https://confluence.atlassian.com/bamboo/bamboo-8-0-release-notes-1077903836.html
https://www.atlassian.com/software/bamboo/download-archives
https://marketplace.atlassian.com/apps/27818/artifactory-for-bamboo?hosting=server&tab=versions

**Comment by eyalbe4** [comment]
@lancelinksp -
Version 2.3.2 of the Bamboo Artifacxtory Plugin has just been marked as compatible with version 8.0.1 of Bamboo.

**Comment by lancelinksp** [comment]
> @lancelinksp -
> Version 2.3.2 of the Bamboo Artifacxtory Plugin has just been marked as compatible with version 8.0.1 of Bamboo.

Assuming you meant version 3.2.3. I see that in the Version History page now, https://marketplace.atlassian.com/apps/27818/artifactory-for-bamboo/version-history, but strange it doesn't show up when you just look at the plugin page initially.

Thanks for the reply

**Comment by eyalbe4** [comment]
Yes @lancelinksp - I meant 3.2.3 - I've just edited my initial comment. 
Did you mean that version 3.2.3 of the plugin doesn't show as compatible with Bamboo 8.0.x from inside the Bamboo UI?

**Comment by lancelinksp** [comment]
I actually meant the marketplace listing. The marketplace listing does show correctly now as well. In Bamboo for me it only shows version 3.2.2 is available since at this moment I am on Bamboo version 7.1.4 but will be upgrading soon and needed to verify compatibility before doing so. Thanks for the help.

**Comment by shuvadipc** [comment]
This is a very old thread and not relevant any more. Closing this.
